.github/workflows/ci.yml

```diff
@@ -109,6 +109,13 @@ jobs:
         if: matrix.os != 'ubuntu-20.04'
         run: yarn run test:integration
 
+      # New version of playwright requires to install new browsers
+      - name: Install Playwright browsers
```

Comment:
Why was this necessary to be added? Just wanting to understand

Reply:
If you check this run on the CI Pipeline, the Playwright itself indicates to install new browsers in order to run correctly:
https://github.com/Lichtblick-Suite/lichtblick/actions/runs/12296062820/job/34314281675

Maybe this can be eliminated in the future, but by now it was the way I found to keep it working smoothly.